ProtectionScan/Features/MainFeature.cs

```diff
@@ -28,7 +28,10 @@ internal sealed class MainFeature : Feature
 
         private const string _fileOnlyName = "file-only";
         internal readonly FlagInput FileOnlyInput = new(_fileOnlyName, ["-f", "--file"], "Print to file only");
-
+        
+        private const string _jsonName = "json";
```

Comment:
This should only be defined for when the framework version is the correct range.